### PR TITLE
Create GitHub Actions workflow for generating the "latest" binaries release

### PR DESCRIPTION
**Pull request by MartinPetkov**
Putting this up as example for discussion later. I do not plan to submit yet.

### CODE REVIEW

File: .github/workflows/release-latest-binaries.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        ./build/install-hub.sh
+
+        # If the release exists already, delete it
+        # For some reason, `hub edit` doesn't seem to update assets as expected
+        if hub release | grep -q 'latest'; then
+          bin/hub release delete 'latest'
+        fi
+
+        # Create the release with all binaries
+        bin/hub release create $(printf -- ' --attach=%s' ./*-amd64) -m 'Binaries latest release' 'latest' --prerelease
```
Comment (xingao267) [inline .github/workflows/release-latest-binaries.yaml:67]: add a comment why it's a prerelease

Comment (MartinPetkov) [reply .github/workflows/release-latest-binaries.yaml:68]: Done.

File: .github/workflows/release-latest-binaries.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      shell: bash
+      run: ./build/build-binaries.sh -v 'latest'
+
+    # From https://github.com/github/hub#github-actions
+    - name: Create GitHub Release
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      shell: bash
+      run: |
+        set -x
```
Comment (umairidris) [inline .github/workflows/release-latest-binaries.yaml:55]: set -ex?

File: .github/workflows/release-latest-binaries.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+        # If the release exists already, delete it so it can be recreated
+        # For some reason, `hub edit` fails to update assets and crashes with various network errors
+        if hub release | grep -q 'latest'; then
+          bin/hub release delete 'latest'
+        fi
+
+        # Create the release with all binaries
+        # Set the pre-release flag to indicate that it is not ready for production and may be unstable
+        bin/hub release create $(printf -- ' --attach=%s' ./*-amd64) -m 'Binaries latest release' 'latest' --prerelease
```
Comment (xingao267) [inline .github/workflows/release-latest-binaries.yaml:68]: let's rename release to build to emphasize it's not a real stable release.

Comment (MartinPetkov) [reply .github/workflows/release-latest-binaries.yaml]: Done.

File: .github/workflows/release-latest-binaries.yaml (hunk 2 above)
Comment (MartinPetkov) [reply .github/workflows/release-latest-binaries.yaml]: Done.

**Pull request merged**